dataland-framework-toolbox/src/main/kotlin/org/dataland/frameworktoolbox/frameworks/eutaxonomynonfinancials/custom/EuTaxonomyAmountWithCurrencyComponentFactory.kt

```diff
@@ -0,0 +1,44 @@
+package org.dataland.frameworktoolbox.frameworks.eutaxonomynonfinancials.custom
+
+import org.dataland.frameworktoolbox.intermediate.components.ComponentBase
+import org.dataland.frameworktoolbox.intermediate.group.ComponentGroupApi
+import org.dataland.frameworktoolbox.intermediate.group.create
+import org.dataland.frameworktoolbox.template.TemplateDiagnostic
+import org.dataland.frameworktoolbox.template.components.ComponentGenerationUtils
+import org.dataland.frameworktoolbox.template.components.TemplateComponentFactory
+import org.dataland.frameworktoolbox.template.model.TemplateRow
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.stereotype.Component
+
+/**
+ * Generates EuTaxonomyAmountWithCurrency from rows with the component "Custom - AmountWithCurrency"
```

Comment:
Either this comment is faulty or the string in canGenerateComponent.

Reply:
Comment is faulty. Fixed it.